.github/workflows/cd-badger.yml

```diff
@@ -1,10 +1,18 @@
 name: cd-badger
-on: workflow_dispatch
+on:
+  workflow_dispatch:
+    inputs:
+      releasetag:
+        description: 'releasetag'
+        required: true
+        type: string
```

Comment:
user entered

Reply:
this workflow is triggered manually via the github UI; and the tag is applied once the release is cut on the github UI

Reply:
essentially release has the steps:
- run cd-dgraph workflow (pipeline); this will build dgraph and create artifacts and push images to dockerhub
- make release on github ui; as part of this step github will automatically apply a tag at the specific commit we are releasing
- upload artifacts to the release on github

Reply:
Where is the tag getting applied? I don't see that step in the workflow.

Reply:
@sanjayk-github-dev - 

the process is as follows:
- cut a tag
- enter the tag from step(1) to trigger the cd pipeline

If I have understood your question correctly - I have not automated the auto-trigger on tag because we have to figure out a process to limit tag creators. If someone accidentally adds a tag, that could trigger the pipeline. The manual step ensures we are safe today. This can evolve.

Reply:
I was wondering if the tag can be created as a step in the workflow once the workflow is successfully completed.

Reply:
Sticking to tag based model, discussed over a call.